enterprise/backend/test/metabase_enterprise/audit_app/pages/dashboard_subscriptions_test.clj

```diff
@@ -35,7 +35,7 @@
                                                                     :collection_id collection-id}
                           PulseChannel          {channel-id :id}   {:pulse_id       pulse-id
                                                                     :channel_type   "email"
-                                                                    :details        {:emails ["amazing@fake.com"]}
+                                                                    :details        {:emails ["amazing@metabase.com"]}
```

Comment:
Just curious, this change is unrelated right? And you're changing it because we want to use domains we control to prevent accidents (if e.g. these tests were run in a context where we could actually send out emails)?

Reply:
Actually, this test was failing for me in the REPL because of the `subscription-allowed-domains` setting in `metabase-enterprise.advanced-config.models.pulse-channel`. It was failing because locally I had an enterprise token set with all of the features enabled, but in CI I guess this test is run without the `:email-allow-list` feature, so the `fake.com` domain doesn't error. 

I thought that `metabase.com` was the default value I think that's actually just set by a different test. A better fix would be to just clear all premium features for the duration of this test, which I actually did in some other tests in this PR as well. (i.e. `(premium-features-test/with-premium-features #{} ...)`) I'll change this to use that approach as well.